action.yml

```
@@ -59,8 +64,14 @@ runs:
             exit 1
           fi
         fi
-
-        helm plugin install https://github.com/helm-unittest/helm-unittest >/dev/null 2>/dev/null
+        helm_ut_plugin_version="${{ inputs.unittest-version }}"
+        if [[ ! -z "${helm_ut_plugin_version// }" ]]; then
```

In keeping with POSIX compliance:

```suggestion
        if [ -n "$helm_ut_plugin_version" ]; then
```

We do lose the ability to "trim" the input, but I think that's okay since the user has to go out of their way to send just whitespace.